docs/advanced-topics/verification.rst

```diff
@@ -0,0 +1,126 @@
+Verifying a test parameter file
+===============================
+
```

Comment:
(to be answered in the docs)
Who is the intended audience for this documentation?
Who will be able to do this?
Why would they want to do this?